setup.py

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env python
-from setuptools import setup, find_packages
+from setuptools import setup, find_namespace_packages
```

Comment:
I think it needs to be `find_packages`? Unless we are now using namespace packages (e.g. `google.subpackageA`, `google.subpackageB`, etc, PEP-420).

Reply:
Hmm, I changed this as it allowed `tox` to more easily find the submodules, it was after an hour of frustration. I've reverted this change and it works, so maybe I changed something else that fixed it! Good pickup :)